src/navs/index.js

```diff
@@ -2706,6 +2706,12 @@ export const docsMenu = {
                     icon: 'IconHogQL',
                     color: 'purple',
                 },
+                {
+                    name: 'Filtering, Visualizing, and Joining Data',
```

Comment:
I think this is fine so it fits in the sidebar, the main article can have the full title.
```suggestion
                    name: 'Joining data',
```